Sprint2

AleksandrMakridin opened the pull request:
Sprint2

AleksandrMakridin committed:
Add files via upload

Sprint 2 (task 1-3)

AleksandrMakridin committed:
Add files via upload

task 5 + 6

AleksandrMakridin committed:
Create .gitkeep

mongo-sharding

AleksandrMakridin committed:
Create .gitkeep

mongo-sharding-repl

AleksandrMakridin committed:
Create .gitkeep

sharding-repl-cache

AleksandrMakridin committed:
Add files via upload

mongo-sharding

AleksandrMakridin committed:
Add files via upload

mongo-sharding-repl

AleksandrMakridin committed:
Add files via upload

sharding-repl-cache

AleksandrMakridin committed:
Update compose.yaml

name: mongo-sharding

AleksandrMakridin committed:
Update compose.yaml

sharding

AleksandrMakridin committed:
Add files via upload

mongodb-countdoc

AleksandrMakridin committed:
Update mongo-init.sh

###
# Инициализируем бд
###

AleksandrMakridin committed:
Update README.md

Mongo-sharding

AleksandrMakridin committed:
Update compose.yaml

mongo-sharding-repl

AleksandrMakridin committed:
Update compose.yaml

pymongo_api:

AleksandrMakridin committed:
Update compose.yaml

mongo-sharding-repl

AleksandrMakridin committed:
Update mongo-init.sh

AleksandrMakridin committed:
Add files via upload

mongodb-countdoc.sh

AleksandrMakridin committed:
Update README.md

AleksandrMakridin committed:
Update compose.yaml

AleksandrMakridin committed:
Update compose.yaml

config-data-configdb:

AleksandrMakridin committed:
Update compose.yaml

  config-data-configdb:

AleksandrMakridin committed:
Update compose.yaml

name: mongo-sharding

AleksandrMakridin committed:
Update compose.yaml

mongo-sharding-repl

AleksandrMakridin committed:
Update compose.yaml

sharding-repl-cache

AleksandrMakridin committed:
Update mongo-init.sh

AleksandrMakridin committed:
Update mongodb-countdoc.sh

AleksandrMakridin committed:
Update mongo-init.sh

AleksandrMakridin committed:
Update mongodb-countdoc.sh

AleksandrMakridin committed:
Update mongo-init.sh

AleksandrMakridin committed:
Update mongo-init.sh

AleksandrMakridin committed:
Create mongodb-countdoc.sh

Create mongodb-countdoc.sh

AleksandrMakridin committed:
Update README.md

README.md

AleksandrMakridin committed:
Update README.md

sprint 2

AleksandrMakridin committed:
Update README.md

Container console

ivannizh commented:
Ошибки при поднятии контейнеров

![image](https://github.com/user-attachments/assets/343b8e04-a43f-438a-91b9-8a907343b222)
![image](https://github.com/user-attachments/assets/9b322f91-f152-4be3-96ce-9b5dd170b1c7)
![image](https://github.com/user-attachments/assets/b2bfef94-52c3-40df-a4ff-d6f28ecd055d)

AleksandrMakridin committed:
Delete spint2alltask.drawio

Delete spint2alltask.drawio

AleksandrMakridin committed:
Add files via upload

Исправление
task 5 Убран nginx, redis исправлен на кластер
task 5 Redis исправлен на кластер

AleksandrMakridin committed:
Update README.md

task 5 Убран nginx, redis исправлен на кластер
task 6 Redis исправлен на кластер

AleksandrMakridin committed:
Update compose.yaml

Исправление круг 1

AleksandrMakridin committed:
Update mongo-init.sh

Оставлена только инициализация

AleksandrMakridin committed:
Update mongo-init.sh

AleksandrMakridin committed:
Update mongo-init.sh

compose

AleksandrMakridin committed:
Update README.md

AleksandrMakridin committed:
Update README.md

AleksandrMakridin committed:
Update compose.yaml

Исправлено название реплик

AleksandrMakridin committed:
Update mongo-init.sh

Поправил разбмвку на shard-ы (1.1 - 2.1)

AleksandrMakridin committed:
Update compose.yaml

shard1 - 2

AleksandrMakridin committed:
Update mongo-init.sh

shard 1 - 2

AleksandrMakridin commented:
Исправлено
Схемы task2 задания 5(убран nginx, redis отражен как кластер ) и 6 (redis отражен как кластер).
Yaml - ы всех 3-х заданий

ivannizh commented:
Новая проблема
![image](https://github.com/user-attachments/assets/1f04a168-1a04-43b4-ae4e-3d40d9460d4e)

AleksandrMakridin commented:
Т.к я натворил делов с мерджем. Подготовил отдельное описание.
Полное описания выполнения 
https://github.com/AleksandrMakridin/architecture-sprint-2/blob/sprint2/Sprint2.docx
mongo-sharding
mongo-sharding-repl
sharding-repl-cache
Скрипты, yaml-ы, скрины в файле
